### ISSUE TITLE
How to debug a specific set of data of a theory?

### ISSUE THREAD

**Issue by tongbong** [xunit/resharper-xunit#112]
Assuming the code below and using resharper as the test runner, resharper shows a test for each individual set of data. 

Unfortunately it is impossible to debug only one set of data, or is there a way to do it? 

```
        public static IEnumerable<object[]> Data()
        {   yield return new object[] { 0, 0 }; // set 1
            yield return new object[] { 0, 1 }  // set 2}

        [Theory]
        [MemberData("Data")]
        public void CheckStructure(int left, int right)
        {
              // Test something
        }
```

**Comment by citizenmatt** [comment]
This is fixed as part of the ReSharper 10.1 support - the runner will now actually run/debug just the tests you ask for.

**Comment by tongbong** [comment]
Brilliant, thanks!
